src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommandRunner.cs

```diff
@@ -33,9 +33,9 @@ public async Task ExecuteCommandAsync(ListPackageArgs listPackageArgs)
         {
             if (!File.Exists(listPackageArgs.Path))
             {
-                Console.Error.WriteLine(string.Format(CultureInfo.CurrentCulture,
-                        Strings.ListPkg_ErrorFileNotFound,
-                        listPackageArgs.Path));
+                await Console.Error.WriteLineAsync(string.Format(CultureInfo.CurrentCulture,
```

Comment:
I'm not sure this has any benefit for us. 
This isn't something that's called concurrently, so there's no benefit to it.